config/sync/core.entity_form_display.taxonomy_term.su_news_tag_filters.default.yml

```diff
@@ -0,0 +1,46 @@
+uuid: fd0a4150-45f7-4578-8114-370fb9906812
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.su_news_tag_filters
+  module:
+    - path
+    - text
+id: taxonomy_term.su_news_tag_filters.default
+targetEntityType: taxonomy_term
+bundle: su_news_tag_filters
+mode: default
+content:
+  description:
```

Comment:
Let's add a description, as long as we are revising. Friendlier for site editors!

"Create filters for use with the News option for Filtered Lists paragraph"